src/Adapter/Cart/CommandHandler/AddCustomizationFieldsHandler.php

```diff
@@ -62,7 +67,7 @@ public function handle(AddCustomizationFieldsCommand $command): int
 
         foreach ($customizationFields as $customizationField) {
             $customizationFieldId = (int) $customizationField['id_customization_field'];
-            //@todo validation
+            //@todo fields validation ?
```

Comment:
What kind of validation ? checking for example that an extra price custom field is indeed a float ?

Reply:
I wasn't sure if we shouldn't somehow improve the legacy validation. As custom text fields seems to have no symbols/length constraints and file validations are also kinda primitive.

Reply:
Can you check how custom text fields are stored in database ?
My guess is they are stored as a standard text field, and then we should apply the same constraints as standard text fields (so 255 max length ?). For symbols, I guess we should leave it free.

For files, I see in `AddCustomizationFieldsHandler` you check size and type (gif, png, jpg). I guess it's enough, isn't it ?